src/scene/renderer/forward-renderer.js

```diff
@@ -415,6 +415,12 @@ class ForwardRenderer {
         this.nearClipId.setValue(camera._nearClip);
         this.farClipId.setValue(camera._farClip);
 
+        if (this.scene.physicalUnits) {
+            this.exposureId.setValue(camera.getExposure());
```

Comment:
Should the scene exposure be ignored here, or should it be somehow added on top of the camera exposure? If ignored, it'd be good to update its docs in the Scene to mention its not used when physicalUnits are used.

Reply:
Done, it's mentioned in the documentation for both the physicalUnits switch as well as the direct exposure value on the camera.